src/ucp/wireup/select.c

```diff
@@ -660,10 +660,16 @@ static double ucp_wireup_rma_score_func(ucp_context_h context,
                                         const uct_iface_attr_t *iface_attr,
                                         const ucp_address_iface_attr_t *remote_iface_attr)
 {
+    uct_ppn_bandwidth_t remote_attr = {
+        .dedicated = remote_iface_attr->bandwidth.dedicated,
```

Comment:
actually, would be better to convert dobule to float and not vice versa
so precision loss would not break symmetry
it means ucp_tl_iface_bandwidth should accept the type defined in address.h (2 floats)